### PR TITLE
Create palindrome.c

### PR DESCRIPTION
**Pull request by christallobo**
This Pull Request resolves Issue #<IssueNumber>

Description:
In this Pull Request we have done the following additions/edits for the code:

Note: Delete all the text including this!
* Make sure you replace `#<IssueNumber>` with the particular Issue that you are referencing with!
* Make sure you describe the Issue properly and ask as many people as possible to review your Pull Request so that it can be merged as soon as possible!

### CODE REVIEW

**Comment by MJ10** [comment]
This is not a part of any issue, please create a new issue on the repository (Refer to #3 for format) and reference that issue in the PR description

File: palindrome/plindrome.c [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
```
Comment (MJ10) [inline palindrome/plindrome.c]: Please rename it to your Github username

**Comment by salman-bhai** [comment]
Pull Request will be closed in 10 hours. Please send required changes before that.

**Comment by DerinAntony** [comment]
Code in C to check if a number is palindrome or not

**Pull request closed without merging**